sycl/doc/extensions/IntelGPU/IntelGPUDeviceInfo.md

```diff
@@ -0,0 +1,239 @@
+# SYCL(TM) Proposal: New device descriptors for Intel GPUs
+
+**IMPORTANT**: This specification is a draft.
+
+**NOTE**: Khronos(R) is a registered trademark and SYCL(TM) is a trademark of the Khronos Group, Inc.
+
+New device descriptors will be added to provide access to low-level hardware details about Intel GPU devices.  This information will be useful to developers tuning specifically for those devices.
+
+This proposal details what is required to provide this information as a SYCL extensions.
+
+## Feature Test Macro ##
+
+The Feature Test Macro will be defined as:
+
+    #define SYCL_EXT_INTEL_DEVICE_INFO 1
+
+
+
+# Intel GPU PCI Address #
```

Comment:
This should not be "Intel GPU" now. Please also review other instances of "Intel GPU" in this document.

Reply:
Just for PCI address?  With the exception of maybe "memory bandwidth", the rest are extremely Intel GPU specific.

Reply:
Done!  I made both PCI address and max memory bandwidth more generic.